api_server.py

```
@@ -74,12 +86,54 @@ def format_hours(minutes: Optional[int]) -> Optional[str]:
     return f"{minutes / 60.0:.2f}h"
 
 
+def format_availability_data(data: Any, now: datetime) -> Dict:
```

Ruff (0.15.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `data`

(ANN401)